ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -726,3 +726,5 @@ def get_current_test_marks():
     not in [VAULT_KMS_PROVIDER, HPCS_KMS_PROVIDER],
     reason="This test requires both Vault or HPCS KMS deployment to be enabled and a valid KMS provider.",
 )
+
+ui = compose(skipif_ibm_cloud_managed, pytest.mark.ui)
```

Comment:
ui is too generic name here. when we see ui marker for test cases, it gives impression that test case is related to UI.

Reply:
@vavuthu we already have this UI marker here:
https://github.com/red-hat-storage/ocs-ci/blob/25d9e7549cb32d8700e8f33f7f12ab2d830d2452/ocs_ci/framework/pytest_customization/marks.py#L74

So I guess @vasudha786 forgot to delete the one from line 74.

On ROKS we do not support UI testing as of now - so I suggested Vasudha to combine it to skip basically all UI decorated tests on ROKS platform instead of separately decorate new and new tests.